swf/src/avm2/write.rs

```diff
@@ -124,12 +124,12 @@ impl<W: Write> Writer<W> {
         Ok(())
     }
 
-    #[allow(dead_code)]
     fn write_i24(&mut self, n: i32) -> Result<()> {
-        // TODO: Verify n fits in 24-bits.
-        self.write_u8(((n >> 16) & 0xff) as u8)?;
-        self.write_u8(((n >> 8) & 0xff) as u8)?;
-        self.write_u8((n & 0xff) as u8)?;
+        let bytes = n.to_le_bytes();
+        debug_assert_eq!(bytes[3], 0);
```

Comment:
Don't think this assert is right because if `n` is negative, this byte will be 0xff, which is valid. Instead can check `n >= -8388608 && n <= 8388607`.

Reply:
If I'm not wrong, this should be equivalent:
```
debug_assert!(bytes[3] == 0 || bytes[3] == 0xFF);
```